src/crypto/CryptoInterface.c

```diff
@@ -617,13 +669,13 @@ int BSL_Crypto_AddRegistryKey(const char *keyid, const uint8_t *secret, size_t s
     string_init_set_str(keyid_str, keyid);
```

Comment:
I think `_set_at()` makes a copy of the keyid. Does this leak the string? Is this covered by a unit test?